asv/plugins/virtualenv.py

```diff
@@ -130,15 +130,15 @@ def _setup(self):
         env = dict(os.environ)
         env.update(self.build_env_vars)
 
-        log.info("Creating virtualenv for {0}".format(self.name))
+        log.info(f"Creating virtualenv for {self.name}")
         util.check_call([
             sys.executable,
             "-mvirtualenv",
             "-p",
             self._executable,
             self._path], env=env)
 
-        log.info("Installing requirements for {0}".format(self.name))
+        log.info("Installing requirements for {self.name}")
```

Comment:
Not an f-string. We need to be careful with that, since it's very likely that not flake8 and not the tests will warn us when we miss the `f` to make an f-string, and we'll lose the value of the variables in the string.

Reply:
noted